Wordle.Api/Data/AppDbContext.cs

```diff
@@ -12,5 +12,6 @@ public AppDbContext(DbContextOptions<AppDbContext> options)
         }
 
         public DbSet<ScoreStat> ScoreStats { get; set; } = null!;
+        public DbSet<Player> PlayerStats { get; set; } = null!;
```

Comment:
```suggestion
        public DbSet<Player> Players{ get; set; } = null!;
```
Typical naming convention is to just make it a plural version of the class's name.